### PR TITLE
Week 4: Libraries -- Add Google Chart API + COVID-19 API

### PR DESCRIPTION
**Pull request by JabariD**
In about.html, get an updated view of COVID-19 statistics in a chart format.

Link: https://paytondennis-step-2020.uc.r.appspot.com/about.html
@ptai7 @chrisc94

### CODE REVIEW

File: portfolio/src/main/webapp/charts/charts.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    google.charts.setOnLoadCallback(drawSeriesChart);
+
+    /** Creates a chart and adds it to the page. */
+    function drawSeriesChart() {
+
+        var data = google.visualization.arrayToDataTable(array);
+
+        const options = {
+            title: 'Most populous countries COVID-19 statistics. The current date is: ' + date,
+            hAxis: {title: 'New Confirmed Cases'},
```
Comment (ptai7) [inline portfolio/src/main/webapp/charts/charts.js:64]: nit: Is this new confirmed cases per day/per week?

File: portfolio/src/main/webapp/charts/charts.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        const date = data.Date;
+
+        let array = [];
+
+        // Push key
+        const key = ['ID', 'New Confirmed Cases', 'New Deaths', 'Total Confirmed', 'Total Deaths'];
+        array.push(key);
+
+        // Countries we're getting data from! These refer to the country codes based on the API.
+        // After 26 added random countries
```
Comment (ptai7) [inline portfolio/src/main/webapp/charts/charts.js:33]: What does this line mean? Is 26 referring to the line number?

File: portfolio/src/main/webapp/charts/charts.js (hunk 1 above)
Comment (JabariD) [reply portfolio/src/main/webapp/charts/charts.js:64]: Yes, we should make that clearer. Should be per day.

File: portfolio/src/main/webapp/charts/charts.js (hunk 2 above)
Comment (JabariD) [reply portfolio/src/main/webapp/charts/charts.js:33]: Oops that should be deleted. Using for testing purposes when working with API.

**Pull request merged**